src/util/ieee_float.cpp

```diff
@@ -1246,7 +1245,7 @@ float ieee_floatt::to_float() const
   CHECK_RETURN(i.is_ulong());
   CHECK_RETURN(i <= std::numeric_limits<std::uint32_t>::max());
 
-  a.i=i.to_ulong();
+  a.i = numeric_cast_v<unsigned>(i);
   return a.f;
```

Comment:
Use uint32_t instead of unsigned?

Reply:
Indeed, and will then remove the above `CHECK_RETURN` as `numeric_cast_v` subsumes these.